go/project_structure/operator.md

```diff
@@ -0,0 +1,59 @@
+# Operator
+
+This page describes common code structure for our operators.
+
+The example below is taken from [aws-operator] on 2019/11/01. Most of the files
+and directories are elided for simplification. Note that not all packages are
+described as this page should be extended over time.
+
+```
+./service/
+├── collector/
+├── controller/
+│   ├── controllercontext/
+│   ├── internal/
```

Comment:
I would like to hide `internal/` since this is only internal but giving an impression as an important component.

Reply:
I think it's important from the code structure standpoint.

Reply:
When we create a new operator, I don't think we would ever need to create `internal/` as first step. But up to you, I'm okay with it.

Reply:
Agreed but I'd like to keep it. It's useful for me during reviews when operators become "more advanced".